### PR TITLE
breaking(curriculum): add probability calculator to /learn

### PR DESCRIPTION
**Pull request by zairahira**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [ ] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md [hunk 1]
```diff
@@ -56,36 +47,209 @@ probability = experiment(hat=hat,
...
+```
+
+# --seed--
+
+## --seed-contents--
+
+```py
+import copy
+import random
+# Consider using the modules imported above.
```
Comment (ShaunSHamilton) [inline curriculum/challenges/english/07-scientific-computing-with-python/scientific-computing-with-python-projects/probability-calculator.md:191]:
```suggestion

```

**Review by ShaunSHamilton — CHANGES_REQUESTED** [review]
We might also want to think about adding runnable examples to the description; examples that include an output, and some indication as to what the output could/should be.

**Comment by ShaunSHamilton** [comment]
> Example usage is already there, line 38 on wards. I hope that is what is being requested.

Right. I was going more for something that prints to the console. Currently, that example does nothing.

That is probably fine, because this project's API is so simple Campers can easily figure out adding `print(hat)`. 👍

**Review by ShaunSHamilton — APPROVED** [review]
<s>LGTM 👍 </s>

Oops: The `# --instructions` needs to be `description`

**Review by ShaunSHamilton — APPROVED** [review]
Failing test is unrelated.

**Review by ihechikara — APPROVED** [review]

**Pull request merged**